Wireframe/css/style.css

```diff
@@ -0,0 +1,128 @@
+:root{
+    --background-color: rgb(43, 42, 42);
+    --text-color: white;
+    --link-color: rgb(64, 126, 226);
+    --article_background:rgb(87, 85, 85);
+
+    
+
+}
+html{
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+body{
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 5px 0 5px 5px;
+
+    background-color: var(--background-color);
+    color: var(--text-color);
+    font-family: 'Courier New', Courier, monospace;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-shadow: 1px 0px black;
+
+}
+ .header-c{
+    width: 60%;
+    text-align: left;
+    justify-content: left;
+    margin-bottom: 20px;
+    padding: 5px;
+    
+    
+ }
+ .header-c h2{
+    text-align: center;
+    font-size: 30px;
+ }
+
+ .section{
+    
+    padding: 20px;
+    width: 75%;
+    
+    background-color: var(--article_background);
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(2 , auto-fit) ;
+    margin: 0 auto;
+    
+ }
+
+ .section1:nth-child(1){
+    display: block;
+    border: 2px solid black;
+    /* align-items: center; */
+    direction: ltr;
+    margin: 0  0 10px 0;
+    padding: 10px;
+
+ }
+
+.section2{
+    flex-wrap: wrap;
+    justify-content: space-between;
+    display: grid;
+    grid-template-columns: repeat(2,45%);
+    grid-gap: 4px; 
+    margin: 0;
+    justify-content: space-between; 
+    
+
+
+    
+
+}
+
+
+
+ .section2>article{
+    
+    
+    
+    border: 2px solid black;
+    padding: 15px;
+
+
+
+ }
+
+ a{
```

Comment:
How does the user know what to click on?  What does WebAIM say about links?

https://webaim.org/techniques/hypertext/link_text